docs/en/Xamarin.Essentials/Launcher.xml

```diff
@@ -133,5 +133,49 @@
         </remarks>
       </Docs>
     </Member>
+    <Member MemberName="TryOpenAsync">
+      <MemberSignature Language="C#" Value="public static System.Threading.Tasks.Task&lt;bool&gt; TryOpenAsync (string uri);" />
+      <MemberSignature Language="ILAsm" Value=".method public static hidebysig class System.Threading.Tasks.Task`1&lt;bool&gt; TryOpenAsync(string uri) cil managed" />
+      <MemberSignature Language="DocId" Value="M:Xamarin.Essentials.Launcher.TryOpenAsync(System.String)" />
+      <MemberType>Method</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>Xamarin.Essentials</AssemblyName>
+        <AssemblyVersion>1.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Threading.Tasks.Task&lt;System.Boolean&gt;</ReturnType>
+      </ReturnValue>
+      <Parameters>
+        <Parameter Name="uri" Type="System.String" />
+      </Parameters>
+      <Docs>
+        <param name="uri">To be added.</param>
```

Comment:
We should fill in these docs. Do you have DocWriter installed?

Reply:
Could we populate these with inline comments?  i'll install docwriter